code/code/misc/statistics.cc

```diff
@@ -268,7 +276,7 @@ void save_game_stats(void)
          gold_modifier[GOLD_SHOP_RESPONSES].getVal(),
          gold_modifier[GOLD_DUMP].getVal());
 
-    fprintf(fp, "%f\n", stats.equip);
+    fprintf(fp, "%f %f %f\n", stats.equip, stats.global_lp_target, stats.global_lp_target_changerate);
```

Comment:
Outputs the new stats to the log now, for persistence across boots.

Reply:
So you _read_ the new format but _write_ the old one?

Reply:
No, it writes to the new format as well. It reads three values, and writes out three values.

Reply:
I just happened to stick a remark on the spot where the read is. Check lines 178-188.